draft-demarco-oauth-status-attestations.md

```diff
@@ -249,11 +307,14 @@ related to a specific Credential issued by the same Credential Issuer.
 +-------------------+                         +--------------------+
 ~~~
 
-The Wallet Instance sends the Status Assertion request to the Credential Issuer.
-The request MUST contain the base64url hash value of the Digital Credential, for which the Status Assertion
-is requested, and enveloped in a signed object as proof of possession.
-The proof of possession MUST be signed with the private key corresponding
-to the public key attested by the Credential Issuer and contained within the Digital Credential.
+The Wallet Instance sends the Status Assertion request to the
+Credential Issuer, where:
+- The request MUST contain the base64url hash value of the Digital Credential,
+for which the Status Assertion is requested, and enveloped in a signed
+object as proof of possession.
+- The proof of possession MUST be signed with the private key corresponding
+to the public key attested by the Credential Issuer and contained within
```

Comment:
```suggestion
to the confirmation claim assigned by the issuer and contained within
```

Reply:
this text generalize the scope of the cnf claim within the jwt/cwt

this makes sense to me